bin/src/Main.purs

```diff
@@ -515,7 +516,7 @@ main =
           Install args@{ packages, selectedPackage, ensureRanges, testDeps } -> do
             { env, fetchOpts } <- mkFetchEnv { packages, selectedPackage, ensureRanges, testDeps }
             -- TODO: --no-fetch flag
-            dependencies <- runSpago env (Fetch.run fetchOpts)
+            packageDependencies <- runSpago env (Fetch.run fetchOpts)
```

Comment:
```suggestion
            dependencies <- runSpago env (Fetch.run fetchOpts)
```

This is a bit of a nit, but after these few iterations I think it would make sense to go back to calling this `dependencies`, mostly for terseness, but also so that we can use `packageDependencies` for when we extract the set of dependencies from a package

Reply:
If `Map PackageName PackageMap` was defined as a type (preferably a newtype per my comment about hating type aliases below), then this value could be named after that type. Not sure what that name would be. Perhaps `packageToDependencies`? Verbose, yes, but still leaves room for `packageDependencies`, `dependencies`. Moreover, it implies a mapping of some sort.

Reply:
I think what I'm getting at is that the content carried by this name has not changed so much to warrant a project-wide rename